app/src/server/api/routers/payments.router.ts

```diff
@@ -0,0 +1,173 @@
+import { z } from "zod";
+import { createTRPCRouter, protectedProcedure } from "~/server/api/trpc";
+import { requireIsProjectAdmin, requireNothing } from "~/utils/accessControl";
+import { error, success } from "~/utils/errorHandling/standardResponses";
+import { TRPCError } from "@trpc/server";
+import {
+  createStripeCustomerAndConnectItToProject,
+  getDefaultPaymentMethodId,
+  getPaymentMethods,
+  getStripeCustomerId,
+  setDefaultPaymentMethod,
+  createSetupIntent,
+  getStripeCustomer,
+  deletePaymentMethod,
+} from "~/server/utils/stripe";
+import { chargeInvoice } from "~/server/tasks/chargeInvoices.task";
+
+export const paymentsRouter = createTRPCRouter({
+  createStripeCustomer: protectedProcedure
```

Comment:
I don't fully grok the flow yet, but will `createStripeCustomer` and `createStripeIntent` always be called at the same time? If so, should they be combined into a single mutation?

Reply:
Yes, it should work fine!